cypress/e2e/39-projects.cy.ts

```diff
@@ -220,12 +211,172 @@ describe('Projects', { disableAutoLogin: true }, () => {
 				.should('contain.text', 'Notion account personal project');
 		});
 
-		it('should move resources between projects', () => {
+		it('should allow to change inaccessible credential when the workflow was moved to a team project', () => {
```

Comment:
```suggestion
		it('should allow changing an inaccessible credential when the workflow was moved to a team project', () => {
```

A better name for the test?